### DISCUSSION TITLE
Marked Private In Outlook Want to show up in Google Calendar

### DISCUSSION THREAD

**Discussion by ElroyBJ** [derekantrican/GAS-ICS-Sync#383] [General]
I want to mark my Appointments as Private in Outlook but want them to show up when moved to Google. Is this possible? I have the var overrideVisibility = "public" set like this and it still shows Private Appointment.

**Comment by jonas0b1011001** [comment]
I assume outlook is not publishing event details if set to private. You can confirm that by downloading the outlook ics, open it in any text editor and check what info is available.

**Comment by jonas0b1011001** [comment]
Closing due to lack of feedback.
